id-library-showcase/Cargo.toml

```diff
@@ -1,37 +1,38 @@
 [package]
-name = "concordium-idlayer-examples"
+name = "id-library-showcase"
 version = "0.1.0"
 edition = "2018"
 license-file = "../LICENSE"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-id = { version = "*", path = "../concordium-rust-sdk/concordium-base/rust-src/id" }
-concordium-rust-sdk = { version = "*", path = "../concordium-rust-sdk/"}
+id = { version = "*", path = "../deps/concordium-rust-sdk/concordium-base/rust-src/id" }
+concordium-rust-sdk = { version = "*", path = "../deps/concordium-rust-sdk/"}
 tokio = { version = "1.5.0", features = ["full"] }
 tonic = "0.4.2"
 anyhow = "1.0"
 serde_json = "1.0.60"
 serde = "1.0"
 rand = "=0.7"
 dialoguer = "0.8"
-crypto_common = { version = "*", path = "../concordium-rust-sdk/concordium-base/rust-src/crypto_common" }
+crypto_common = { version = "*", path = "../deps/concordium-rust-sdk/concordium-base/rust-src/crypto_common" }
 pairing = "0.15"
+rpassword = "5.0"
 
 structopt = "0.3"
 clap = "2.33.3"
 
 [dependencies.pedersen_scheme]
-path = "../concordium-rust-sdk/concordium-base/rust-src/pedersen_scheme"
+path = "../deps/concordium-rust-sdk/concordium-base/rust-src/pedersen_scheme"
 version = "0"
 
 [dependencies.curve_arithmetic]
-path = "../concordium-rust-sdk/concordium-base/rust-src/curve_arithmetic"
+path = "../deps/concordium-rust-sdk/concordium-base/rust-src/curve_arithmetic"
 version = "0"
```

Comment:
This also is partially exported. If more is needed than the traits please re-export that.